marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -211,6 +211,8 @@ class Jobs {
       }
       docObj.batch.error = `${error.name || error.code}: ${error.message}`;
     }
+    docObj.batch.writeTrnxID = writeTransactionInfo.transaction;
+    docObj.batch.writeTimeStamp = writeTransactionInfo.dateTime;
```

Comment:
is this wallclock time too?

Reply:
no

Reply:
Could you make it a wallclock timestamp?

We should be consistent with the timestamps in the docs

Reply:
sorry, I misread. It is wallclock. It uses fn.currentDateTime, since xdmp.requestTimestamp returns an empty sequence in update transactions.

Reply:
ah right.